frontend/src/Editor/AppVersionsManager/ReleasedVersionError.jsx

```diff
@@ -1,11 +1,24 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import cx from 'classnames';
+import { useAppVersionStore } from '@/_stores/appVersionStore';
+import { shallow } from 'zustand/shallow';
 
-export const ReleasedVersionError = ({ isUserEditingTheVersion, changeBackTheState }) => {
-  React.useState(() => {
+export const ReleasedVersionError = () => {
+  const { isUserEditingTheVersion, disableReleasedVersionPopupState } = useAppVersionStore(
+    (state) => ({
+      isUserEditingTheVersion: state.isUserEditingTheVersion,
+      disableReleasedVersionPopupState: state.actions.disableReleasedVersionPopupState,
+    }),
+    shallow
+  );
+  const changeBackTheState = useCallback(() => {
+    isUserEditingTheVersion && disableReleasedVersionPopupState();
+  }, [isUserEditingTheVersion, disableReleasedVersionPopupState]);
+
+  React.useEffect(() => {
```

Comment:
Instead of using `React.useEffect` we can import the `useEffect` directly and use it for better readability. We can follow this as a standard.

Reply:
Done.